_includes/manuals/1.20/1.2_release_notes.md

```diff
@@ -419,12 +419,68 @@ Foreman can now run properly in FIPS-enabled environments.
 
 *A full list of changes in {{page.version}} is available via [Redmine](https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f[]=cf_12&op[cf_12]=%3D&v[cf_12][]=885)*
 
+### Release notes for 1.20.1
+#### API
+* 'subtotal' is incorrect in API responses ([#25487](https://projects.theforeman.org/issues/25487))
+
+#### Audit Log
+* Audits contain no data (Added foo to Missing(ID: x)) ([#25437](https://projects.theforeman.org/issues/25437))
+
+#### Compute resources
+* compute resource host import does not correctly set the compute ressource ([#25513](https://projects.theforeman.org/issues/25513))
```

Comment:
ressource -> resource (fixed in Redmine)

Reply:
fixed